File: submissions/Utkarsh Saxena/day15_maze_sum.c
#include <stdio.h>
#include <string.h>
#include <stdlib.h>
int main(){
    char input[2100];
    int sum = 0;

    printf("Enter numbers: ");
    fgets(input, sizeof(input), stdin); //read input including spaces
    
    char *token = strtok(input, " ");
    while(token != NULL){
        sum += atoi(token); //convert toekn to int and add to sum
        token = strtok(NULL, " ");
    }
    printf("Secret Code: %d\n", sum); //print final result

    return 0;
    
}

File: submissions/Ayush_Vishwakarma/day15_maze_sum.py
# Take input from the user, split the input by spaces, convert each to an integer, and store in a list
l = list(map(int, input("Enter numbers : ").split()))

# Initialize a variable to store the sum of numbers
sum = 0

# Loop through each number in the list and add it to the sum
for i in l:
    sum += i

# Print the final sum as a "Secret code"
print("Secret code :", sum)
